__tests__/db.test.js

```diff
@@ -120,6 +120,86 @@ describe('app', () => {
            })
           })
         })
+        test('POST - api/articles/:article_id/comments - checks that extra inputs are ignored', () => {
+          const commentToAdd = {
+            username: 'butter_bridge',
+            body: "I haven't posted in a while!",
+            extraProperty: 'extraProperty'
+          };
+          return request(app)
+          .post('/api/articles/3/comments')
+          .send(commentToAdd)
+          .expect(201)
+          .then((res) => {
+           expect(res.body.comment.hasOwnProperty('extraProperty')).toBe(false)
+           })
+        })
+        test('PATCH - api/articles/:article_id - responds with 200 & updated votes property', () => {
+          return request(app)
+          .patch('/api/articles/1')
+          .send({
+            inc_votes: 15
+          })
+          .expect(200)
+          .then((res) => {
+            expect(res.body.updatedVote).toMatchObject({
+              article_id: 1,
+              title: 'Living in the shadow of a great man',
+              topic: 'mitch',
+              author: 'butter_bridge',
+              body: 'I find this existence challenging',
+              votes: 115,
+            })
+          })
+        })
+        test('PATCH - api/articles/:article_id - responds with 200 & updated votes property for minus votes', () => {
+          return request(app)
+          .patch('/api/articles/1')
+          .send({
+            inc_votes: -15
+          })
+          .expect(200)
+          .then((res) => {
+            expect(res.body.updatedVote).toMatchObject({
+              article_id: 1,
+              title: 'Living in the shadow of a great man',
+              topic: 'mitch',
+              author: 'butter_bridge',
+              body: 'I find this existence challenging',
+              votes: 85,
+            })
+          })
+        })
+        test('PATCH - api/articles/:article_id - responds with 200 & updated votes property for minus votes when current vote is 0', () => {
```

Comment:
Could probably reword this, is it checking that votes can't go into minus numbers?